Programs

TopPoptart opened the pull request.

TopPoptart committed:
Delete Standard Assets.meta

TopPoptart committed:
Delete World1.unity.meta

TopPoptart committed:
Delete World1.unity

TopPoptart committed:
Delete Start Screen.csproj

TopPoptart committed:
Delete New Terrain.asset.meta

TopPoptart committed:
Delete New Terrain.asset

TopPoptart committed:
Delete New Terrain 1.asset.meta

TopPoptart committed:
Delete New Terrain 1.asset

TopPoptart committed:
Delete First villager.blend

TopPoptart committed:
Create Start Screen .csproj

rylanwest committed:
Delete Start Screen .csproj

Alicekiller2 committed:
World 1 Update  3/22/18

Gradual update

rylanwest committed:
Add files via upload

rylanwest committed:
Delete Start Screen.unity

rylanwest committed:
Delete Game Screen.unity

rylanwest committed:
Delete MainMenu.cs

rylanwest committed:
Delete MainMixer.mixer

rylanwest committed:
Delete New TMP Color Gradient.asset

rylanwest committed:
Delete SettingsMenu.cs

rylanwest committed:
Delete Start Screen.csproj

rylanwest committed:
Delete Start Screen.sln

rylanwest committed:
Add files via upload

CaptainMarvel15 committed:
Update Rigged Villager

rylanwest committed:
removed extra files

rylanwest committed:
Merge branch 'master' into Programs

rylanwest committed:
upload start screen

rylanwest commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Menu Screen](https://trello.com/c/f4CU6K54/5-menu-screen)

CaptainMarvel15 committed:
3/26 Updated Villager

Rigged but arms are not connected for some reason. Head bone is fixed and works.

CaptainMarvel15 committed:
Male Villager Updated

Bones are all connected and the model is able to move around fine

CaptainMarvel15 committed:
Male Villiage IK and Root Bone Fixed and added

Buka401 committed:
Female villager

Cycles render mode and texture applied
Texture is in trello

CaptainMarvel15 committed:
Male Villager fully Rigged, Ready to animate

CaptainMarvel15 committed:
Villiager Walk Cycle Started

unable to flip poses

CaptainMarvel15 committed:
Finished Male Villiager

Don't know how to import into our Unity Project

CaptainMarvel15 committed:
Finished Wolf Walk Cycle

needs to be imported into Unity

TopPoptart committed:
Add files via upload

rylanwest committed:
Merge branch 'master' into Programs

mrsimonsen commented:
old
